nikola/plugins/command/new_post.py

```diff
@@ -213,6 +213,14 @@ class CommandNewPost(Command):
                     'bbcode, html, textile, txt2tags',
         },
         {
+            'name': 'kernel',
```

Comment:
It should totally fail when used with other compilers. And it should say in the help that it's only important for jupyter, something like

'help': 'Kernel to use for jupyter posts, irrelevant for all other formats' or somesuch